### ISSUE TITLE
App asks for full keyboard input access for all apps - that is just too much

### ISSUE THREAD

**Issue by coopr** [miho/SonoAir#35]
When launching beta 7 on MacOS 10.15.13, the app requests permission to access all keystrokes always. That is way too much privacy to delegate to an app that should only be streaming audio.

Also, it does work at all (never discovers anything), but the keyboard monitoring issue is the first issue to solve :)

**Comment by trietsch** [comment]
I agree, requesting to access all keystrokes made me uninstall this app.

**Comment by joetague** [comment]
Saw this comment in the middle of `brew cask install sonoair`, so `ctrl+c` to cancel - no way does it need access to keystrokes.

**Comment by jameslinnell** [comment]
Agreed, serious security issue, doesn't leave me with much confidence so it's uninstalled until fixed.
